qlog-metainfo.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/res/io.github.foldynl.QLog.metainfo.xml b/res/io.github.foldynl.QLog.metainfo.xml
+index f76414d..be71e37 100644
+--- a/res/io.github.foldynl.QLog.metainfo.xml
++++ b/res/io.github.foldynl.QLog.metainfo.xml
+@@ -46,5 +46,9 @@
+   <url type="contact">https://github.com/foldynl</url>
+   <url type="vcs-browser">https://github.com/foldynl/QLog</url>
+   <url type="contribute">https://github.com/foldynl/QLog/blob/master/CONTRIBUTING.md</url>
++  <releases>
++    <release version="0.30.0" date="2023-12-01"/>
```

Comment:
@foldynl The releases section needs to be updated every time a new release is out. It would be great if we could do this directly in upstream (instead of having to manually update this downstream patch every time) before making a new QLog release if possible. Ideally not manually, but by using some automated script.

Reply:
The release section was not a part of your original QLog's [PR](https://github.com/foldynl/QLog/pull/262/files).
ufff...thats was my question [here](https://github.com/foldynl/QLog/issues/264#issuecomment-1817857190) Point 2 and your response was: "We have something called "external-data-checker" on Flathub that automatically checks for new releases (GitHub tags etc.) and creates Pull requests that update the Flatpak manifest file. So, ideally just downloading/installing and quick testing (I can do that if you wish) of the test build that will automatically get created in such PR could be sufficient. If there will be a problem, I can handle it or give you a hint, no worries."

so now i'm confused. QLog metainfo file should newly contains this section ? it's a little uncomfortable

Reply:
OK..just to be sure. I have to add this section to QLog's metainfo file. The release section should contain all releases and their release date. The section must be updated every release. Correct?

Reply:
Correct.

Or do not add the section and it will have to be patched (and the patch manually updated) in the Flathub repo as it is now.

Reply:
.